app/navigation/NotificationNavigator.scala

```diff
@@ -52,6 +52,11 @@ class NotificationNavigator @Inject()() {
       case None => routes.OffshoreLiabilitiesController.onPageLoad(NormalMode)
     }
 
+    case WhatIsYourFullNamePage => ua => ua.get(WhatIsYourFullNamePage) match {
+      case Some(_) => routes.YourPhoneNumberController.onPageLoad(NormalMode)
+      case None => routes.WhatIsYourFullNameController.onPageLoad(NormalMode)
+    }
+
```

Comment:
Where we don't care what the data is that we retrieve we can simplify this to:
`case WhatIsYourFullNamePage => _ => routes.YourPhoneNumberController.onPageLoad(NormalMode)`

Reply:
fixed..